serpentTools/parsers/microxs.py

```diff
@@ -0,0 +1,271 @@
+"""Parser responsible for reading the ``*mdx[i].m`` files"""
```

Comment:
This file needs to be formatted according to our [coding style](http://serpent-tools.readthedocs.io/en/latest/develop/codestyle.html). More comments in review comment